lib/Module/Build.pm

```diff
@@ -93,13 +93,12 @@ the "./" notation, you can do this:
 C<Module::Build> is a system for building, testing, and installing
 Perl modules.  It is meant to be an alternative to
 C<ExtUtils::MakeMaker>.  Developers may alter the behavior of the
-module through subclassing in a much more straightforward way than
-with C<MakeMaker>.  It also does not require a C<make> on your system
-- most of the C<Module::Build> code is pure-perl and written in a very
-cross-platform way.
+module through subclassing.  It also does not require a C<make> on your
+system - most of the C<Module::Build> code is pure-perl and written in a
+very cross-platform way.
 
-See L<"MOTIVATIONS"> for more comparisons between C<ExtUtils::MakeMaker>
-and C<Module::Build>.
+See L<"COMPARISON"> for more comparisons between C<Module::Build> and
```

Comment:
`L<"...">` for section linking is an older style that has been removed from the spec, although it is still supported by most formatters.  It should be changed to `L</...>`.